concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -2304,6 +2305,132 @@ jobs:
         params:
           release: pipeline-pool
 
+  - name: reset-cloudfoundry-passwords
```

Comment:
Agreed. I think because it is low risk and low severity we don't need to do anything about it.